Dockerfile-windows.dapper

```diff
@@ -1,4 +1,4 @@
-FROM rancher/golang:1.14-windowsservercore
+FROM rancher/golang:1.15-windowsservercore
```

Comment:
@luthermonson how do I get this new image pushed to https://hub.docker.com/r/rancher/golang/tags?

Reply:
https://github.com/rancher/golang and will need a new pr to add an 1.15 dir, do 1.16 while youre there... or.. if oyu dont need this... just avoid it and stick with 1.14 and ill do em all later

Reply:
I'll just remove the last commit since we don't need Golang to be bumped for the Dockerfile-windows.dapper (since it's only involved in running `docker build` that pulls in binaries, not actually building those binaries)